tests/Community/Tdarr_Plugin_AV1_HandBreak_Transcode.js

```diff
@@ -0,0 +1,185 @@
+/* eslint no-plusplus: ["error", { "allowForLoopAfterthoughts": true }] */
```

Comment:
You don't paste the entire plugin into the tests folder, please look at other tests.

Reply:
I'm looking at other tests but I don't grasp how it should be written or when to use the required files. Sorry, I'm still fairly new to programming.